Stake command to control snapshots flag (#2211)
nucypher/nucypher: docs/source/guides/network_node/staking_guide.rst

    (nucypher)$ nucypher stake winddown --hw-wallet

Snapshots
*********

> Taking snapshots is *enabled* by default. This option gives ability to be involved in DAO. To stop taking snapshots:

Reviewer: ```suggestion
Taking snapshots is *enabled* by default. Snapshots must be enabled to participate in the DAO. To stop taking snapshots:
```

> Taking snapshots is *enabled* by default. Snapshots must be enabled to participate in the DAO. To stop taking snapshots:

Reviewer 2: Maybe mention why a staker may be interested in disabling snapshots, which is basically to have a slight reduction on gas expenditure.

Reviewer 2: ```suggestion
Taking snapshots is *enabled* by default. Snapshots must be enabled to participate in the DAO, but it has a slight cost in gas every time your staking balance changes. To stop taking snapshots:
```

Author: Ok, I'll add today but later

.. code:: bash

>     (nucypher)$ nucypher stake snapshots --disable

Reviewer: @vzotova is there a reason someone would want to disable snapshots?

Author: Less gas spending on `mint` and `commitToNextPeriod`, iirc 25k gas less

Reviewer: 👍 - make sense. If you don't care about participating in the DAO, you save on some gas costs.
